perllib/FixMyStreet/Cobrand/CentralBedfordshire.pm

```diff
@@ -170,6 +170,8 @@ sub open311_extra_data_include {
     ];
 
     if ($contact->email =~ /Jadu/) {
+        my $reported_by_staff = $row->get_extra_metadata('contributed_as', 'n/a') eq 'another_user' ? 'Yes' : 'No';
```

Comment:
Is fine, but I wonder if they'd want to have staff-as-anonymous-user or -body come through as staff as well? Might be easier to see if `contributed_by` is set and go on that?